src/ecdsa/keys.py

```diff
@@ -222,21 +472,112 @@ def to_string(self, encoding="raw"):
         else:
             return self._compressed_encode()
 
-    def to_pem(self):
-        return der.topem(self.to_der(), "PUBLIC KEY")
+    def to_pem(self, point_encoding="uncompressed"):
+        """
+        Convert the public key to the :term:`PEM` format.
+
+        The PEM header of the key will be `BEGIN PUBLIC KEY`.
+
+        The format of the key is described in the `from_der()` method.
+        This method supports only "named curve" encoding of keys.
+
+        :param str point_encoding: specification of the encoding format
+            of public keys. "uncompressed" is most portable, "compressed" is
+            smallest, "hybrid" is uncommon and unsupported by most
+            implementations, is as big as "uncompressed". "raw" will create
```

Comment:
Why is "raw" allowed if it generates malformed files?

Reply:
because there is no code to forbid it, filed #136 to fix that (I want to keep this PR docs and stylistic changes only)